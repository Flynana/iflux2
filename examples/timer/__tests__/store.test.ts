import Store from '../src/store'

describe('store', () => {
  it('store init state', () => {
    const store = new Store({})
    const state = store.state()
    expect(state).toMatchSnapshot()
  })

  it('store increment', () => {
    const store = new Store({})
    store.dispatch('start')
    expect(store.state()).toMatchSnapshot()
  })

  it('store decrement', () => {
    const store = new Store({})
    store.reset()
    expect(store.state()).toMatchSnapshot()
  })
})