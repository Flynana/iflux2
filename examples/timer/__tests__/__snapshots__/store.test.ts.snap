exports[`store store decrement 1`] = `
Object {
  "time": 0,
}
`;

exports[`store store increment 1`] = `
Object {
  "time": 1,
}
`;

exports[`store store init state 1`] = `
Object {
  "time": 0,
}
`;
